src/core/util/ajax.js

```diff
@@ -1,5 +1,5 @@
 // @ts-check
-/* eslint-disable no-unused-vars */
+
 import progressbar from '../render/progressbar.js';
```

Comment:
Would prefer no unused cars (and not adding ignore comments but actually removing them)

Reply:
ESLint's `no-unused-vars` is configured in `eslint.config.js`. The comment is auto-removed when running `lint:fix` because it is unnecessary. The rule is still enforced.